### ISSUE TITLE
How do I manipulate child data?

### ISSUE THREAD

**Issue by simasware** [crocodic-studio/crudbooster#1498]
I would like to manipulate data from child before inserting/updating but I can't using $postdata. How can I do that?

**Comment by Gopal-Bandyopadhyay** [comment]
Yes this is a serious problem. I did not find any solution to manipulate child data before insert the database even i did catch post child data at anywhere.
But it is noticed, Child data insert at Hook_after_add function.
CBController file at vendor\crocodicstudio\crudbooster\src\controllers\CBController.php may have sone clue. please see
